README.md

```diff
@@ -27,19 +27,20 @@ This action can only be run after a Terraform `fmt`, `init`, `plan` or `validate
     TF_WORKSPACE: stage
   with:
     commenter_type: fmt/init/plan/validate # Choose one
-    commenter_input: ${{ format('{0}{1}', steps.step_id.outputs.stdout, steps.step_id.outputs.stderr) }}
```

Comment:
I would suggest to keep `commenter_input` in the example and add `commenter_input_file` as option

Reply:
I would argue that since this is the more stable option to suggest `commenter_input_file` as default. `commenter_input` is only there to stay backwards compatible for people using this action as suggested
with `Jimdo/terraform-pr-commenter@main`. 🤔 we could introduce versioning and make it a breaking change.

Reply:
It turns out versioning is complicated as
- our runners do like to run on this repository
- we are not able to change the default branch
so for now we will go ahead with the current PR and maybe later cleanup by introducing versioning and breaking changes.